pages/styles/style.css

```diff
@@ -0,0 +1,141 @@
+body {
+    background-color: #FEBCAE;
+    color:#4120A9;
+}
+
+#index #about {
+    display: flex;
+    Flex-Direction: Column;
```

Comment:
Google Chrome is smart enough to know you meant `flex-direction: column;` but be sure to double check property and value names to be certain they're the correct case.